assets/bootstrap_5/build/css/bootstrap_5-rtl.css

```diff
@@ -10927,4 +10927,14 @@ a {
 
 .table > :not(:first-child) {
   border-top: 0;
+}
+
+.gender-button li {
```

Comment:
Why gender button ? Why not ls-button-item since we have Yes/No and theme using button too …

If we add a css class for each input inside each question type : create new theme takes week …